src/Workspaces/CSharp/Portable/Formatting/Engine/Trivia/TriviaDataFactory.Analyzer.cs

```diff
@@ -59,7 +59,7 @@ public static AnalysisResult Between(SyntaxToken token1, SyntaxToken token2)
                     // newline.  If so, we keep track of that so we'll appropriately indent later
                     // on. 
 
-                    var previousNonMissingToken = token1.GetPreviousToken(includeZeroWidth: false, includeSkipped: true);
```

Comment:
No.  The way this works is we get the previous normal token, then we find the last *trivia* of that token.  This is what we want (and I was the one who wrote the comment).  

So even if you have a skipped token, we'll still appropriately get any trivia after that skipped token, because the skipped token will belong to the previous *real* token for the token we have.